docker/README.md

```diff
@@ -258,6 +260,30 @@ You can now start both tunnels:
 ngrok start jetpack jetpack-sftp
 ```
 
+### Docker ngrok
+
+Alternative to the above configuration file is running ngrok in the container with docker-compose file. That starts docker inside a container and you don't have to install it or configure as a standalone software on your machine.
+
+**1. Configure environment**
+
+Add these variables to your `docker/.env` file:
+
+This configures `artpijet.eu.ngrok.io` reserved domain that is available on my basic plan.
+```
+NGROK_AUTH=<your auth key>
+NGROK_SUBDOMAIN=artpijet
+NGROK_REGION=eu
+```
+
+**2. Start docker with additional file**
+
+Instead of starting docker with `yarn docker:up`, you will have to use `docker-compose -f docker/docker-compose.yml -f docker/docker-compose-ngrok.yml up`.
```

Comment:
Should we create a shorthand for this in NPM scripts?

https://github.com/Automattic/jetpack/blob/538b3e7f3877a6e3937493096701d965a6781681/package.json#L26-L27

Perhaps it could be `yarn docker:up-ngrok` or more generic `yarn docker:up-tunneled`:

```
"docker:up-tunneled": "yarn docker:compose -f docker/docker-compose-ngrok.yml up",
```

Reply:
See, this is something I need genuine advice about :)
I did not put it there, since there are scripts dependant on `yarn docker:compose` and I didnt want to introduce the second option, because the dependant ones would not inherit the proper command.

I think your proposition makes the most sense, since  `yarn:docker:up-ngrok` would be the most common use case.
I'll do it!

Reply:
Done!